### PR TITLE
Fixing broken lift/textile/widgets dependencies

### PR DESCRIPTION
**Pull request by csaltos**
Minor changes for the project compile with tuned lift/textile/widgets versions. Not warrantied everything is working, but at least, it's compiling and you can access the project examples using a web browser.

By the way. Thanks for Lift, it's AWESOME !! :) :)

### CODE REVIEW

File: project/build/LiftExamplesProject.scala [hunk 1]
```diff
@@ -42,8 +42,8 @@ class LiftExamplesProject(info: ProjectInfo) extends ParentProject(info) with Li
 
     lazy val lift_facebook = "net.liftweb" %% "lift-facebook" % liftVersion
     lazy val lift_scalate  = "net.liftweb" %% "lift-scalate" % liftVersion
-    lazy val lift_textile  = "net.liftweb" %% "lift-textile" % liftVersion
-    lazy val lift_widgets  = "net.liftweb" %% "lift-widgets" % liftVersion
+    lazy val lift_textile  = "net.liftweb" %% "lift-textile" % "2.4"
```
Comment (fmpwizard) [inline project/build/LiftExamplesProject.scala]: Hi, thanks for the pull request, while it compiles, we are supposed to use a diff version, could you update your pull req to use

`lazy val lift_textile  = "net.liftweb" %% "lift-textile_2.5" % "1.3"`

and

`lazy val lift_widgets  = "net.liftweb" %% "lift-widgets_2.5" % "1.3"`

When we released Lift 2.5, we moved the modules to another repository and now they have independent versions from lift, but as we release newer versions of Lift, we also have to publish diff versions of each module, in short, lift 2.5 has modules with names ending on `_2.5`, and then you can find the "module" version on sonatype, like:
https://oss.sonatype.org/content/repositories/releases/net/liftmodules/textile_2.5_2.10/1.3/

If you could update your pull request with this, add a comment and I'll merge it in.
Thanks and we are glad you like Lift :)

Comment (csaltos) [inline project/build/LiftExamplesProject.scala]: Hello Diego,

It's done. I've had to update some imports in the source code too and the
dependencies towards lift-webkit for the nuggets samples.

There are some compilation warnings that mainly point out a deprecated use
of option boxes that we can attempt to fix in next updates if you agree.

I'm requesting a pull now with these updates.

Thanks again for your support.

Kind regards,

Carlos Saltos

**Comment by fmpwizard** [comment]
Looks like we only have scalate builds for lift 2.6, I'll merge this pull request an then send an email to the list to see if we can get scalate for 2.5 built.

Thanks!

**Comment by csaltos** [comment]
Great !! (Y) :) ... Thanks to you Diego.

Cheers,

Carlos

**Pull request merged**